includes/class-maera-restaurant-styles.php

```diff
@@ -47,105 +47,32 @@ function __construct() {
 		public static function color_palettes() {
 
 			if ( 0 == get_theme_mod( 'invert_palettes', 0 ) ) {
+
 				$palettes = array(
-					'red' => array(
-						'#F5F5F5',
-						'#F9BAAF',
-						'#FB9D8C',
-						'#FD8069',
-						'#FF6347',
-						'#333333',
-					),
-					'orange' => array(
-						'#F5F5F5',
-						'#F9CDAF',
-						'#FBB98D',
-						'#FDA56A',
-						'#FF9148',
-						'#333333',
-					),
-					'yellow' => array(
-						'#F5F5F5',
-						'#F9E6AF',
-						'#FBDF8D',
-						'#FDD86A',
-						'#FFD148',
-						'#333333',
-					),
-					'green' => array(
-						'#F5F5F5',
-						'#C9F7C4',
-						'#B3F9AC',
-						'#9DFA94',
-						'#88FC7C',
-						'#333333',
-					),
-					'blue' => array(
-						'#F5F5F5',
-						'#D9E3F6',
-						'#CCDAF7',
-						'#BED1F8',
-						'#B1C9F9',
-						'#333333',
-					),
-					'violet' => array(
-						'#F5F5F5',
-						'#E5E0F6',
-						'#DED6F6',
-						'#D6CCF7',
-						'#CFC2F8',
-						'#333333',
-					)
+					'red'    => array( '#F5F5F5', '#F9BAAF', '#FB9D8C', '#FD8069', '#FF6347', '#333333' ),
+					'orange' => array( '#F5F5F5', '#F9CDAF', '#FBB98D', '#FDA56A', '#FF9148', '#333333' ),
+					'yellow' => array( '#F5F5F5', '#F9E6AF', '#FBDF8D', '#FDD86A', '#FFD148', '#333333' ),
+					'green'  => array( '#F5F5F5', '#C9F7C4', '#B3F9AC', '#9DFA94', '#88FC7C', '#333333' ),
+					'blue'   => array( '#F5F5F5', '#D9E3F6', '#CCDAF7', '#BED1F8', '#B1C9F9', '#333333' ),
+					'violet' => array( '#F5F5F5', '#E5E0F6', '#DED6F6', '#D6CCF7', '#CFC2F8', '#333333' )
 				);
+
 			} else {
+
 				// Array indexes still match non inverted names.
 				$palettes = array(
-					'red' => array(
-						'#00B7AF',
-						'#058088',
-						'#085C67',
-						'#093848',
-						'#DDDDDD',
-					),
-					'orange' => array(
-						'#0076BA',
-						'#005B98',
-						'#00448D',
-						'#001E5E',
-						'#DDDDDD',
-					),
-					'yellow' => array(
-						'#182A9C',
-						'#03077E',
-						'#01044E',
-						'#000131',
-						'#DDDDDD',
-					),
-					'green' => array(
-						'#C47979',
-						'#864265',
-						'#572457',
-						'#300F61',
-						'#DDDDDD',
-					),
-					'blue' => array(
-						'#8C5D42',
-						'#794B30',
-						'#603D29',
-						'#472F21',
-						'#DDDDDD',
-					),
-					'violet' => array(
-						'#667047',
-						'#3E4C2B',
-						'#1B2205',
-						'#101A05',
-						'#DDDDDD',
-					)
+					'red'    => array( '#00B7AF', '#058088', '#085C67', '#093848', '#DDDDDD' ),
```

Comment:
short and consistent arrays can be written in the same line... shorter and easier to read.